source/Playnite/Localization/LocSource.xaml

```diff
@@ -717,6 +731,10 @@ More information at https://playnite.link/cefstartup</sys:String>
     <sys:String x:Key="LOCHideGame">Hide this game</sys:String>
     <sys:String x:Key="LOCUnHideGame">Remove from Hidden</sys:String>
     <sys:String x:Key="LOCEditGame">Edit…</sys:String>
+    <sys:String x:Key="LOCCalculateInstallSize">Calculate install size</sys:String>
+    <sys:String x:Key="LOCCalculateGamesAllInstallSize">Calculate install size (All games)</sys:String>
+    <sys:String x:Key="LOCCalculateGamesMissingInstallSize">Calculate install size (Only missing data)</sys:String>

```

Comment:
I don't think this option will be needed anymore if we start storing last calculation date for each game separately. Then we can just automatically skip calculation for up to date values.

Reply:
Marking as resolved as discussed on Discord

![image](https://user-images.githubusercontent.com/1389286/191129078-9fa72b88-108a-459a-9c5b-2f438e07ec5e.png)